feat: add contributing guide (#175)
FuelLabs/docs-hub: docs/contributing/documentation.mdx

---
title: Documentation
category: Contributing
parent:
  label: All Docs
  link: /
---

# Documentation Content

All documentation, with the exception of the guides and `intro` section, is pulled in to the `docs-hub` repo via git submodules.

All contributions to the content of the documentation should be done in the original repository.

> You can see a list of where the source documentation for each book lives below:

Reviewer: ```suggestion
Below, you will find a list indicating the locations of the source documentation for each book:
```
